client_test.go

```diff
@@ -6428,7 +6428,8 @@ func Test_NewClient_Defaults(t *testing.T) {
 	require.False(t, enqueuer.StaggerStartupIsDisabled())
 
 	reindexer := maintenance.GetService[*maintenance.Reindexer](client.queueMaintainer)
-	require.Equal(t, []string{"river_job_args_index", "river_job_metadata_index"}, reindexer.Config.IndexNames)
+	require.Contains(t, reindexer.Config.IndexNames, "river_job_args_index")
+	require.Contains(t, reindexer.Config.IndexNames, "river_job_metadata_index")
```

Comment:
Want to add assertions for the others too?

Reply:
I guess I was mostly thinking it'd be good to have it be complete for if/when we allow customizing the list, but that's a preemptive concern.